Upgrade Nextra

AleCandido opened the pull request:
Closes #19 

I would suggest merging after #21 (instead of *in* #21) in order not to pollute that PR.

- [x] upgrade package version
- [x] check result
- [x] drop the embedded (outdated) guide
- [x] fix configs

AleCandido commented:
@scarlehoff whenever you can have a look, that just means try to run in development mode (possibly on MacOS, so much the better) and check that everything still looks as before!

scarlehoff commented:
Looks ok. At least I don't see anything wrong reading through the vrap tutorial.

AleCandido commented:
> Looks ok. At least I don't see anything wrong reading through the vrap tutorial.

There was a small hiccup in the style of the logo, that for some reason was not applying Tailwind classes. I reverted to classic CSS, now it should be good.
(and another tiny one in which the reference to the home was appearing explicitly, also fixed...)
